dsl.md

```diff
@@ -466,20 +474,21 @@ document:
   namespace: default
   name: customFunctionWorkflow
   version: '0.1.0'
-
 do:
   - validateEmail:
-      call: https://github.com/myorg/functions/validateEmailAddress@v1
+      call: https://raw.githubusercontent.com/myorg/main/validateEmailAddress/1.0.0
```

Comment:
FWIW, `https://github.com/myorg/functions/validateEmailAddress@v1` also works nicely. Users can tag their functions on their private repos instead of a direct link to the raw file. It's elegant.

Reply:
I'l replace it then, so that we have both examples!